lib/commands/web.js

```diff
@@ -17,6 +17,8 @@ const TAB_BAR_OFFSET = 33;
 const IPHONE_WEB_COORD_SMART_APP_BANNER_OFFSET = 84;
 const IPAD_WEB_COORD_SMART_APP_BANNER_OFFSET = 95;
 
+const CALIBRATION_TAP_DELTA = 5; // px
```

Comment:
```suggestion
const CALIBRATION_TAP_DELTA = 7; // px
```

When this value was 7, my local iPad had the best score in landscape/portrait, and when the side bar was shown. I tested a few website such as https://www.yahoo.co.jp/ to click when a target element was edge of the bottom and right for example. Occasionally still i saw a wrong click when the target element was small, although this pr should be good as an initial implementation I believe

Reply:
I checked a few on simulators with this value `7` as well, that worked.

Reply:
this is good news. Changed it to 7, also altered the documentation regarding nativeWebTapStrict capability